### ISSUE TITLE
CUSA01130 - Gravity Rush Remastered

### ISSUE THREAD

**Issue by PistolaoBR** [shadps4-emu/shadps4-game-compatibility#1742]
### Game Name

Gravity Rush Remastered

### Game code

CUSA01130

### Game version

v01.00

### Used emulator's version (only released versions are acceptable)

v0.5.0

### Current status

Playable

### Operating System

Windows

### Error

[Core] <Error> stubs.cpp:CommonStub:45: Stub: Unknown (nid: Smf+fUNblPc) called, returning zero to 0x91fffc8db

### Description

The game needs the normal sysmodules and more 3 files libSceFont.sprx, libSceFontFt.sprx, libSceFreeTypeOt.sprx to fix the missing TEXT (Thanks @GHU7924 for the tip: https://github.com/shadps4-emu/shadps4-game-compatibility/issues/75#issuecomment-2562010627 )

@Hermiten @Xphalnos Can I post libSceFont.sprx, libSceFontFt.sprx, libSceFreeTypeOt.sprx here? To help the community

### Screenshots

![image](https://github.com/user-attachments/assets/736576c4-d82a-4e85-9cd8-b38a454dd196)
![image](https://github.com/user-attachments/assets/72ae224d-1621-43e0-9a8d-2f769b2f7717)
![image](https://github.com/user-attachments/assets/e1d22eb7-8deb-41e5-8db3-9933168d3450)
![image](https://github.com/user-attachments/assets/759f80b8-9c28-493c-86aa-eaea55085951)
![image](https://github.com/user-attachments/assets/b43e3e64-e933-4627-94ba-a4a5950d29a3)

### Log File

[shad_log.txt](https://github.com/user-attachments/files/18282700/shad_log.txt)

**Comment by Xphalnos** [comment]
What do you mean by post?

**Comment by PistolaoBR** [comment]
> What do you mean by post?

put the 3 files here... Since the ps4 firmware can be downloaded at the playstation site...

**Comment by Xphalnos** [comment]
People are supposed to know that this file is needed since the necessary modules are given at the shadPS4 Game Compatibility homepage.

**Comment by PistolaoBR** [comment]
> People are supposed to know that this file is needed since the necessary modules are given at the shadPS4 Game Compatibility homepage.

Yes, but in that case, they are not part of the 9 sysmodules files that the shadps4 compatibility page asks, they are another 3... but I wont put it if you guys don't give green light... I do undertand that can lead to some legal problems with playstation

**Comment by Xphalnos** [comment]
The three files you tell me about (libSceFont.sprx, libSceFontFt.sprx and libSceFreeTypeOt.sprx) are already part of the main readme.

**Comment by xXoanon** [comment]
There seems to be an issue with crashing during some combat encounters, has there been a fix found for that yet?

**Comment by PistolaoBR** [comment]
> The three files you tell me about (libSceFont.sprx, libSceFontFt.sprx and libSceFreeTypeOt.sprx) are already part of the main readme.

Thanks... I will read it now

**Comment by PistolaoBR** [comment]
> There seems to be an issue with crashing during some combat encounters, has there been a fix found for that yet?

do you have all 13 files on your sys modules folder

**Comment by xXoanon** [comment]
Yep, all 13 files are there. I've even tried the nightly release, still has the same issue.

`[Tty] <Info> logger.cpp:log_flush:60: [stdout] enemy = grn_s_01
[Tty] <Info> logger.cpp:log_flush:60: [stdout] enemyname = grn_s_01   enemyscore = 10      score = 120
[Render.Vulkan] <Info> vk_pipeline_cache.cpp:CompileModule:478: Compiling fs shader 0xb4adb7b9 (permutation)
[Render.Vulkan] <Info> vk_pipeline_cache.cpp:CompileModule:478: Compiling fs shader 0xb4adb7b9 (permutation)
[Debug] <Critical> liverpool_to_vk.h:PromoteFormatToDepth:80: Unreachable code!`

Here are the last few line in the log after it crashes. The crash always happens at some point during the first combat challenge in he game. I'm not sure if it crashes at any other point in the game after the combat challenge.

**Comment by azurefan01** [comment]
@xXoanon In my experience it can crash every time  you are in combat. I had this happen in combat challenges and in chapters 9 to 18 and in some of the other chapters as well, it's a really weird crash since it doesn't always happen, I played chapter 13 or 14 (don't exactly remember which one) without any crashes in combat and then had a crash in the next chapter's combat section. Fortunately the game autosaves quite frequently and it hasn't crash on any boss for me so far.

**Comment by xXoanon** [comment]
> In my experience it can crash every time you are in combat.
Perhaps the issue has something to do with enemies spawning? The crash happens at a random point in combat, and usually never in the same place as the other times it crashes, so I don't think its a specific enemy or anything like that causing the crash.

**Comment by PistolaoBR** [comment]
Happy new Year mates @Xphalnos @Hermiten

**Comment by Xphalnos** [comment]
> Happy new Year @Xphalnos @Hermiten

To you too and to everyone present here!

**Comment by franfrijol** [comment]
how can i install and play gravity rush on shadps4? i tried in vita3k but in diferent parts you need use the touchscreen, i tried use script but isnt funtional;(

**Comment by Xphalnos** [comment]
> how can i install and play gravity rush on shadps4? i tried in vita3k but in diferent parts you need use the touchscreen, i tried use script but isnt funtional;(

You need to dump it from your PS4

**Comment by Neurasthenya** [comment]
It boots, it plays, it sounds fine with the sys modules, but I do get consistent crashes in challenges ( _sometimes_ I get past the first time just fine, but then it crashes if I try again) and in Episode Three as soon as I start the camera gets all twisty and the screen goes either all orange or all black (only displaying the HUD and the menus) as if it is... stuck in geometry or something? As soon as I activate gravity powers, the camera goes behind Kat's and that's it, cannot play it anymore.

**Comment by Neurasthenya** [comment]
> @Neurasthenya If you are using latest main instead of GR Hacks build (https://github.com/rainmakerv3/shadPS4/releases/tag/GRhack-release) right now there is the issue that motion controls can't be deactivated so the camera might be behaving oddly because of the gyro, in my experience the implementation doesn't behave properly when the controller doesn't have gyro, there is also the fact that it is WIP right now (gyro I mean). The crashes in combat are normal right now, it seems to be a new kind of crash related to the enemy spawn but we don't know for sure yet, since at this time no one has drooped additional information from debbuging. Everything else should be fine I just finished the game entirely in ShadPS4 from 0% to credits.

Ohhhh, I forgot that gyro was a thing for this game, probably that's the issue then, I'm on the main nightly builds

**Comment by jorgecotrinax1** [comment]
![Image](https://github.com/user-attachments/assets/c1759492-9053-410b-8081-fc045b637ac6)  
I have an error with "Gravity Rush remastered", it only happens when activating gravity, everything else works fine

**Comment by magenta7744** [comment]
> There seems to be an issue with crashing during some combat encounters, has there been a fix found for that yet?

Hi how's it going?
Have you found any solutions for this crash during combat issue ?

**Comment by Infiniteking21** [comment]
responding to the previous guys advice about gyro, I don't think that is it. Ive tried disabling that but it seems to still crash.

**Comment by PistolaoBR** [comment]
> > There seems to be an issue with crashing during some combat encounters, has there been a fix found for that yet?
> 
> Hi how's it going? Have you found any solutions for this crash during combat issue ?

Last 0.5.0 nightly fixed that here... see if 0.6.0 or 0.6.1 did it too...

**Comment by Fourknif** [comment]
I'm having a few audio issues when playing this game.
-Audio is quieter in the right speaker than the left speaker.
-Music and most game sounds are very low, even with music and sound effects maxed out in settings.
-Some audio sounds are much louder, namely low gravity gauge warning and gem collection noises.
Anyone else having these issues?

**Comment by moqneko** [comment]
> > > There seems to be an issue with crashing during some combat encounters, has there been a fix found for that yet?
> > 
> > 
> > Hi how's it going? Have you found any solutions for this crash during combat issue ?
> 
> Last 0.5.0 nightly fixed that here... see if 0.6.0 or 0.6.1 did it too...

the latest 0.5.0 build has zero crashes for me but there are no gyro controllers and the fps is very stuttery compared to 0.6.0 and above which crashes in combat for some reason

**Comment by tnuhnivad** [comment]
Can confirm working on v.6 w/ 6.57 sys modules. appears to be fully playable

**Comment by Neurasthenya** [comment]
Working on latest main v.6

- All sys modules, no apparent audio or graphic issues

- No errors with gyro/camera _after putting the in game gyro slider all the way down_, standard xbox generic pad. Menus however cannot be navigated with d-pad due to speed.

- No crashes in extended combat session, however I did got a error pop screen, but was in the middle of pressing a button so the pop up got dismissed, was able to kept playing without crashes 

- Framerate unstable, Win 11 2h22 5600g + 5700xt

[shad_log.txt](https://github.com/user-attachments/files/18775244/shad_log.txt)
